spec.html

```diff
@@ -4178,8 +4178,8 @@ <h1>The Reference Record Specification Type</h1>
           </tr>
           <tr>
             <td oldids="sec-getreferencedname,ao-getreferencedname">[[ReferencedName]]</td>
-            <td>a String, a Symbol, or a Private Name</td>
-            <td>The name of the binding. Always a String if [[Base]] value is an Environment Record.</td>
+            <td>an ECMAScript language value or a Private Name</td>
+            <td>The name of the binding. Always a String if [[Base]] value is an Environment Record. May temporarily be an ECMAScript language value other than a String or a Symbol in the case of a property reference for which ToPropertyKey has yet to be performed.</td>
```

Comment:
I think a sentence of the form "May be an ECMAScript language value [...] until yada yada" would be clearer than "May temporarily be [...]".

Reply:
Sounds good; added an `Otherwise,` to your suggestion to clarify that this just applies to property reference cases.